src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -11,45 +11,194 @@
 
 namespace Symfony\Bridge\Monolog\Formatter;
 
-use Monolog\Formatter\LineFormatter;
+use Monolog\Formatter\FormatterInterface;
 use Monolog\Logger;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Cloner\Stub;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
 
 /**
  * Formats incoming records for console output by coloring them depending on log level.
  *
  * @author Tobias Schultze <http://tobion.de>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class ConsoleFormatter extends LineFormatter
+class ConsoleFormatter implements FormatterInterface
 {
-    const SIMPLE_FORMAT = "%start_tag%[%datetime%] %channel%.%level_name%:%end_tag% %message% %context% %extra%\n";
+    const SIMPLE_FORMAT = "%datetime% %start_tag%%level_name%%end_tag% <comment>[%channel%]</> %message%%context%%extra%\n";
+    const SIMPLE_DATE = 'H:i:s';
+
+    private static $levelColorMap = array(
+        Logger::DEBUG => 'fg=white',
+        Logger::INFO => 'fg=green',
+        Logger::NOTICE => 'fg=blue',
+        Logger::WARNING => 'fg=cyan',
+        Logger::ERROR => 'fg=yellow',
+        Logger::CRITICAL => 'fg=red',
+        Logger::ALERT => 'fg=red',
+        Logger::EMERGENCY => 'fg=white;bg=red',
+    );
+
+    private $options;
+    private $cloner;
+    private $outputBuffer;
+    private $dumper;
+
+    /**
+     * {@inheritdoc}
```

Comment:
This should be removed as you don't extend anything anymore. Also, the phpdoc should document the available option keys.